lib/swc-to-babel.js

```diff
@@ -75,16 +77,16 @@ function toBabel(node, source = '') {
         TSIndexedAccessType,
         TSAsExpression,
         
+        JSXElement,
+        JSXFragment,
+        
         enter(path) {
             const {node} = path;
             const {type} = node;
             
             if ('span' in path.node)
                 convertSpanToPosition(path, source);
             
-            delete node.start;
-            delete node.end;
```

Comment:
Maybe let's keep this two lines?

Reply:
the babel ast has actually

Reply:
why do you need it? How you using it?

Reply:
no use actually, but the project aimed at convert swc ast to babel so I keep this, if you think i should be removed, i can remove it later

Reply:
I think that should be use case for feature that is broken in swc, other things are good to be merged

Reply:
should i remove it ?

Reply:
I've made it back to the original.